assets/src/dashboard/app/views/exploreTemplates/content/stories/index.js

```diff
@@ -35,7 +35,7 @@ const search = {
 };
 const view = {
   style: VIEW_STYLE.GRID,
-  pageSize: { width: 210, height: 316 },
+  pageSize: { width: 212, height: 318, containerHeight: 376.89 },
```

Comment:
Would it be worth making these values constants since they appear across so many files and have meaning?

Reply:
_yes_ i will add them to the storybookUtils